_data/gaming.yml

```
@@ -203,6 +203,13 @@ websites:
     twitter: MojangSupport
     img: mojang.png
 
+  - name: Nexon America
```

From what I see there's nothing different between the login systems for the US version or any other version.
```suggestion
  - name: Nexon
```

---

I can't confirm for other regions, unfortunately. If using "Nexon" as a whole instead is fine, sure.